srcpkgs/pinephone-uboot/template

```diff
@@ -0,0 +1,110 @@
+# Template file for 'pinephone-uboot'
+pkgname=pinephone-uboot
+version=0.0.20200805
+revision=1
+archs="aarch64*"
+create_wrksrc=yes
+hostmakedepends="flex cross-or1k-none-elf dtc python3 python3-devel bc swig"
+depends="u-boot-tools"
+conf_files="/etc/default/${pkgname}-config"
+short_desc="U-Boot for Pinephone with Crust support"
+maintainer="John Sullivan <jsullivan@csumb.edu>"
+license="GPL-2.0-only, BSD-3-Clause"
+homepage="http://www.denx.de/wiki/U-Boot/WebHome"
+
+# FOR MAINTAINERS:
+#
+# This u-boot package relies on downstream support for the pinephone hardware
+# from PINE64 and the Crust maintainers. Until this support is upstreamed,
+# please get sources for any new versions from PINE64's u-boot repo and Crust's
+# branch of arm-trusted-firmware.
+
+_commit_uboot=f0b298c504daae9da171bac950c036432cc00201
+_commit_armtf=f928897fa5df6afa0f70bb1b1011e4bb03cb4596
```

Comment:
I was using crust's arm TF fork because it has optional patches for improved compatibility with crust, but I can probably include these patches as part of the package. Either way, I'll try to get upstream uboot with the pine patches. I believe this is what archlinux arm does as well.

Reply:
So I tried to use upstream u-boot like I said, but it made 20+ patches so I stuck with the downstream repo. (See review thread before this one)

However, crust's version of armtf only had 5 patches, so the template now uses armtf from ARM Software upstream. Patches from crust are applied at crust's branch point ((commit before d0f999c)[https://github.com/crust-firmware/arm-trusted-firmware/commits/crust]), as it's more recent than ARM's latest release (April 20th).